source/agora/node/FullNode.d

```diff
@@ -466,4 +486,69 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    public void pushBlock (const ref Block block) @safe
+    {
+        foreach (BlockExternalizedHandler handler; block_handlers)
+        {
+            runTask({
+                try
+                {
+                    // API POST: /block_externalized
+                    handler.pushBlock(block);
+                }
+                catch (Exception e)
+                {
+                    log.error(
+                        "Error sending block height #", block.header.height
+                        ," to \"", this.config.node.address, "\": ", e);
```

Comment:
This is wrong. It's not sending the block to itself.

Reply:
Also in the future you can use this:

```
` to"`
```

This is simpler than using `\"`.

Reply:
It's fixed now.